src/components/NewGalaxyModal.js

```diff
@@ -286,13 +285,24 @@ export const NewGalaxyModal = _.flow(
   }
 
   const renderPersistentDiskSection = () => {
-    const gridStyle = { display: 'grid', gridTemplateColumns: '0.75fr 4.5rem 1fr 5.5rem 1fr 5.5rem', gridGap: '1rem', alignItems: 'center' }
     return div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
       div({ style: styles.headerText }, ['Persistent disk']),
       div({ style: { marginTop: '0.5rem' } }, [
         'Persistent disks store analysis data.'
         // TODO Add info on PDs for Galaxy (similarly to in NewRuntimeModal.js)
       ]),
+      renderPersistentDiskSizeSection()
+    ])
+  }
+
+  const renderPersistentDiskSizeSection = () => {
+    const gridStyle = { display: 'grid', gridTemplateColumns: '0.75fr 4.5rem 1fr 5.5rem 1fr 5.5rem', gridGap: '1rem', alignItems: 'center' }
+    return currentDataDisk ? div({ marginTop: '0.75rem' }, [div({ style: { ...gridStyle, marginTop: '0.5rem' } }, [
+      div({ style: styles.label }, ['Size (GB): ']), h(TooltipTrigger,
+        { content: ['Persistent disk of size ', currentDataDisk.size, ' GB already exists and will be attached upon Galaxy creation'] }, [
+          div({ marginTop: '0.5rem', width: '5rem' }, [currentDataDisk.size])
+        ])
+    ])]) :
```

Comment:
i feel like line 305 indicates line 300 should be spaced out across a few lines. You could use a `Utils` method to get out of using a ternary (like `Utils.cond`), which could help clean it up